easybuild/easyblocks/p/parmetis.py

```diff
@@ -166,20 +181,30 @@ def install_step(self):
         # other applications depending on ParMETIS (SuiteSparse for one) look for both ParMETIS libraries
         # and header files in the Lib directory (capital L). The following symlink are hence created.
         try:
-            llibdir = os.path.join(self.installdir, 'Lib')
-            os.symlink(libdir, llibdir)
-            for f in ['metis.h', 'parmetis.h']:
-                os.symlink(os.path.join(includedir, f), os.path.join(libdir, f))
+            caplibdir = os.path.join(self.installdir, 'Lib')
+            remove_file(caplibdir)
+            symlink(libdir, caplibdir)
+            for header_file in ['metis.h', 'parmetis.h']:
+                header_path = os.path.join(libdir, header_file)
+                remove_file(header_path)
+                symlink(os.path.join(includedir, header_file), header_path)
         except OSError as err:
             raise EasyBuildError("Something went wrong during symlink creation: %s", err)
 
     def sanity_check_step(self):
         """Custom sanity check for ParMETIS."""
 
+        parmetis_libs = [os.path.join('lib', 'libmetis.a')]
+        # Add static and shared libs depending on configopts
+        if hasattr(self, "config_shared") and self.config_shared is True:
```

Comment:
@lexming no need for `hasattr` as long as you initialize `self.config_shared` in the constructor, done in https://github.com/lexming/easybuild-easyblocks/pull/1